AElf.Database/DatabaseType.cs

```diff
@@ -1,45 +1,50 @@
-﻿using System;
-using System.Collections.Generic;
-
-namespace AElf.Database
-{
-    public enum DatabaseType
-    {
-        KeyValue,
-
-        Redis,
-
-        Ssdb
-    }
-
-    public static class DatabaseTypeHelper
-    {
-        private static readonly Dictionary<string, DatabaseType> _typeDic = new Dictionary<string, DatabaseType>();
-
-        static DatabaseTypeHelper()
-        {
-            _typeDic.Add("keyvalue", DatabaseType.KeyValue);

```

Comment:
We should not hardcode _typeDic like these since the DatabaseType could be modified later.